lib/index.js

```diff
@@ -329,6 +320,16 @@ async function _verifyCredential(options = {}) {
     return result;
   }
 
+  // run common credential checks (add check results to log)
+  _checkCredential({credential, log, now});
+
+  // if credential status is provided, a `checkStatus` function must be given
+  if(credential.credentialStatus && typeof options.checkStatus !== 'function') {
```

Comment:
Would it make sense to move this check to be fore signature verification?

Reply:
Idea was that the first motivation of a verification should always be checking the proof. If this is skipped for whatever reason (some method of field of the credential missing) we would exit without knowing if the credential has been tampered with. Not sure if this is the best approach.